DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/user/user.service.js

```diff
@@ -0,0 +1,35 @@
+﻿(function() {
+    'use strict';
+    angular
+        .module('app.user')
+        .factory('userService', userService);
+
+    userService.$inject = ['$cookies', '$http', 'API_URL'];
+
+    /* @ngInject */
+    function userService($cookies, $http, API_URL) {
+        function login() {
+            location.href = API_URL.LOGIN;
+        }
+
+        function logout() {
+            $cookies.remove('userName');
+            $http.post(API_URL.LOGOUT, {});
+        }
+
+        function userName() {
+            return $cookies.get('userName');
+        }
+
+        function isAuthenticated() {
+            return !!$cookies.get('userName');
```

Comment:
This will fail if the name of our user is an empty string :) hope it is impossible. :)

But I'd better connect `isAuthenticated` to the WEB API auth cookie, not to the name. 

How do you think?